standard/interfaces.md

```diff
@@ -436,138 +426,64 @@ A virtual method with implementation declared in an interface may be overridden
 
 ### 18.4.4 Interface events
 
 This clause augments the description of events in classes [§15.8](classes.md#158-events) for events declared in interfaces.
 
-Interface events are declared using *event_declaration*s ([§15.8.1](classes.md#1581-general)).
-
-*event_modifier* shall not include `override`.
-
-A derived interface may implement an abstract interface event declared in a base interface (§15.8.5).
+Interface events are declared using *event_declaration*s ([§15.8.1](classes.md#1581-general)), with the following additional rules:
 
-An *event_declaration* that has an *event_accessor_declarations* is a default implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract. An *event_declaration* that has no default implementation is always considered abstract; it is *never* considered to be an automatically implemented field-like event ([§15.8.2](classes.md#1582-field-like-events)).
-
-It is a compile-time error for *variable_declarators* in an instance *event_declaration* to contain any *variable_initializer*s.
-
-The type of an interface event shall be input-safe.
+- *event_modifier* shall not include `override`.
+- A derived interface may implement an abstract interface event declared in a base interface (§15.8.5).
+- It is a compile-time error for *variable_declarators* in an instance *event_declaration* to contain any *variable_initializer*s.
+- An instance event with the `virtual` or `sealed` modifiers must declare accessors. It is *never* considered to be an automatically implemented field-like event ([§15.8.2](classes.md#1582-field-like-events)).
+- An instance event with the `abstract` modifier must not declare accessors.
+- The type of an interface event shall be input-safe.
 
 ### 18.4.5 Interface indexers
 
 This clause augments the description of indexers in classes [§15.9](classes.md#159-indexers) for indexers declared in interfaces.
 
-```ANTLR
-interface_indexer_declaration
-    : attributes? 'new'? type 'this' '[' parameter_list ']'
-      '{' interface_accessors '}'
-    | attributes? 'new'? ref_kind type 'this' '[' parameter_list ']'
-      '{' ref_interface_accessor '}'
-    ;
-```
-
-Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)).
+Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)), with the following additional rules:
 
-*indexer_modifier* shall not include `override`.
+- *indexer_modifier* shall not include `override`.
+- An *indexer_declaration* that has a *block* as an *accessor_body* is an implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract.
```

Comment:
Same question about expression-bodied properties and expression-bodied accessors.